x-pack/plugins/observability/public/pages/cases/configure_cases.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+
+import { EuiButtonEmpty } from '@elastic/eui';
+import * as i18n from '../../components/app/cases/translations';
+import { CASES_APP_ID, CASES_OWNER } from '../../components/app/cases/constants';
+import { useKibana } from '../../utils/kibana_react';
+import { useGetUserCasesPermissions } from '../../hooks/use_get_user_cases_permissions';
+import { usePluginContext } from '../../hooks/use_plugin_context';
+import { casesBreadcrumbs, useBreadcrumbs } from '../../hooks/use_breadcrumbs';
+import { getCaseUrl, useFormatUrl } from './links';
+
+const ButtonEmpty = styled(EuiButtonEmpty)`
+  display: block;
+`;
+function ConfigureCasesPageComponent() {
+  const {
+    cases,
+    application: { navigateToApp },
+  } = useKibana().services;
+  const userPermissions = useGetUserCasesPermissions();
+  const { ObservabilityPageTemplate } = usePluginContext();
+  const onClickGoToCases = useCallback(
+    async (ev) => {
+      ev.preventDefault();
+      return navigateToApp(`${CASES_APP_ID}`);
+    },
+    [navigateToApp]
+  );
+  const { formatUrl } = useFormatUrl(CASES_APP_ID);
+  const href = formatUrl(getCaseUrl());
+  useBreadcrumbs([{ ...casesBreadcrumbs.cases, href }, casesBreadcrumbs.configure]);
+  if (userPermissions != null && !userPermissions.read) {
```

Comment:
I think `userPermissions == null` when the security plugin is disabled, right?

Reply:
Oh good point, let me try it real quick.

Reply:
So I tried printing out the permissions in security solution and it looks like when the page first loads it is `null` and then returns `{crud: true, read: true}` when the security plugin is disabled. Any ideas why there's a period of time when the value is `null` I assume that's because it's the initial state in the hook before `useEffect` got run?

Reply:
guys you wrote this code, lol. whatever you want to do is fine with me, i just assumed it worked since it was merged to master. bad assumption?

Reply:
Oh yeah you are right. If the security solution is disabled those values will be true. I think you are right. It has to do with the initial state in the hook.